### PR TITLE
Feature Form

### PR DESCRIPTION
**Pull request by rodrigopenayo98**
Added a new form section in the portfolio.

### CODE REVIEW

**Review by bobb-Rob — COMMENTED** [review]
Hi @rodrigopenayo98, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the contribution of your partner, kindly ensure that your partner(s) contribute to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by rodrigopenayo98** [comment]
@bobb-Rob Hi! Sorry but really... I don't understand it, if you see in the repository you have 2 colaborators me and Umuhan, what do you need? She help me in one commit, you can check it in the commit history... or you need the link of the repository? I had sent you the one with the pull request with my form. Sorry but I dont understand it.

File: index.html [hunk 1]
```diff
@@ -64,7 +61,186 @@ <h4>LET'S CONNECT</h4>
...
+                    <textarea type="textarea" class="textarea" cols="30" rows="10" placeholder="Write your message here..." maxlength="500" required></textarea>
+                  </li>
+                  <li>
+                    <button class="button-form">Get in touch</button>
+                  </li>
+                </ul>
+            </form>
       </section>
       <section class="section4">
         <h2>Contact me</h2>
```
Comment (ABDULALI3468) [inline index.html:246]: ✅You accidently added this section as the contact form is added above already please consider removing this section entirely to meet the project requirements ✅

File: index.html [hunk 2]
```diff
@@ -64,7 +61,186 @@ <h4>LET'S CONNECT</h4>
...
+                    <input class="email" type="email" placeholder="loremipsum@gmail.com"  required>
+                  </li>
+                  <li>
+                    <textarea type="textarea" class="textarea" cols="30" rows="10" placeholder="Write your message here..." maxlength="500" required></textarea>
+                  </li>
+                  <li>
+                    <button class="button-form">Get in touch</button>
+                  </li>
+                </ul>
+            </form>
```
Comment (ABDULALI3468) [inline index.html:243]: On submission of the form, it shows the following error message please consider reviewing that as the form is not even submitted.

### ERROR MESSAGE:-

image

> I request the next reviewer to again check the validations when the form is submitted 💯

File: index.html [hunk 3]
```diff
@@ -64,7 +61,186 @@ <h4>LET'S CONNECT</h4>
...
+                  <li>
+                    <input class="email" type="email" placeholder="loremipsum@gmail.com"  required>
+                  </li>
+                  <li>
+                    <textarea type="textarea" class="textarea" cols="30" rows="10" placeholder="Write your message here..." maxlength="500" required></textarea>
+                  </li>
+                  <li>
+                    <button class="button-form">Get in touch</button>
+                  </li>
+                </ul>
```
Comment (ABDULALI3468) [inline index.html:242]: Please consider making the same padding inside the input elements to make the design exactly in accordance with the Figma design.

### YOUR DESIGN:-

image


### FIGMA DESIGN:-

image

File: index.html [hunk 4]
```diff
@@ -64,7 +61,186 @@ <h4>LET'S CONNECT</h4>
...
+            <img class="li-img" src="./image/Union.png"/>  
+          </ul>
+          <ul class="categories">
+            <li class="li-categories">Skills</li>
+            <img class="li-img" src="./image/Union.png"/>
+          </ul>
+        </div>
+      </section>
+      <section class="section4">
+          <h2>Contact me</h2>
```
Comment (ABDULALI3468) [inline index.html:225]: Please consider adding the same margin-top to the section as used in the Figma design to meet the project requirements.

```
    margin-top: 176.85px;
```

File: styles.css [hunk 5]
```diff
@@ -47,11 +47,33 @@ body {
...
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-weight: 400;
+  width: 327px;
+  font-size: 20px;
+  line-height: 28px;
+  color: #fff;
+  margin-top: 24px;
+}
```
Comment (ABDULALI3468) [inline styles.css:75]: Please consider using the same color as used in the Figma design to make it look alike to the design.


### Color code:-

```
.section4 p {
  color: #EBEBFF;
}
```

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @rodrigopenayo98,



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
☑️Well implemented design.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @rodrigopenayo98  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**